staging/src/k8s.io/apimachinery/pkg/util/yaml/decoder_test.go

```diff
@@ -22,12 +22,26 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
+	"io/ioutil"
 	"math/rand"
 	"reflect"
 	"strings"
 	"testing"
 )
 
+func TestYAMLDecoder(t *testing.T) {
```

Comment:
Name this test to indicate that you are testing the return value of num bytes read.